person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  @id = rand(100)
+
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Your `@id` value must be inside `def initialize(..)` please make it look like below and position like below 🙏 

```ruby
 def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```